### PR TITLE
Support shallow clone [#37]

### PR DESCRIPTION
**Pull request by sharathprabhal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "author": "Greg Bergé <berge.greg@gmail.com>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/neoziro/grunt-shipit/issues"
+    "url": "https://github.com/sharathprabhal/grunt-shipit/issues"
```
Comment (neoziro) [inline package.json]: please :)

**Comment by neoziro** [comment]
Can you fix the issue un package.json and [make an interactive rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i/) with all of your commits to just have one single commit?

```
git rebase -i master
# squash all and rewrite the first one
git push --force
```

**Comment by neoziro** [comment]
@sharathprabhal thank you for that!

**Comment by sharathprabhal** [comment]
Can't believe I did it again. Please take a look at
https://github.com/neoziro/grunt-shipit/pull/42

**Comment by neoziro** [comment]
@sharathprabhal you can do it in the same PR, for the next time try it, just push force and it will work. I close this one and I merge the other.

**Pull request closed without merging**